docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -68,6 +70,33 @@ To use it, add the following to your project's dependencies:
 implementation("io.quarkus:quarkus-webjars-locator")
 ----
 
+=== From a local directory
+
+Static resources can be served from a local directory by installing an additional route in the Vert.x router.
+
+For instance, to serve resources from the `static/` directory relative to the current path at http://localhost:8080/static/,
+you can install the following route:
+
+[source,java]
+----
+package org.acme;
+
+import io.quarkus.runtime.StartupEvent;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.StaticHandler;
+import jakarta.enterprise.event.Observes;
+
+
+public class StaticResources {
+
+    void installRoute(@Observes StartupEvent startupEvent, Router router) {
+        router.route()
+                .path("/static/*")
+                .handler(StaticHandler.create("static/"));
```

Comment:
Do we need to set an `order()` here @cescoffier?

Reply:
Depends if you route conflicts with other routes. Here because it used /static, I don't believe there will be any conflict.